pkg/filesystem/path/builder.go

```diff
@@ -53,6 +64,12 @@ func (b *Builder) GetUNIXString() string {
 	if b.absolute {
 		prefix = "/"
 	}
+	if b.driveletter != rune(0) {
+		// Drive letter is not meaningful in the REAPI. We just drop it.
+		// Symlinks may point to the same drive but not other drives,
+		// so it can be implicit.
+		prefix = "/"
+	}
```

Comment:
Instead of doing this, can't we just let `OnDriveLetter()` set `absolute = true`?